packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3189,10 +3214,10 @@ export class ContainerRuntime
 				const trace = Trace.start();
 
 				const versions = await this.storage.getVersions(
-					null,
+					versionId,
 					1,
 					"refreshLatestSummaryAckFromServer",
-					FetchSource.noCache,
+					versionId === null ? FetchSource.noCache : FetchSource.default,
```

Comment:
If versionId !== null, maybe just pass undefined so driver can use whatever its default scheme is?